prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -14,35 +14,68 @@ identifier_1:
   constants:
     sys: "urn:id:extID"
 
-identifier_2:
+identifier_2a:
   condition: $valueIn NOT_NULL
   valueOf: datatype/Identifier_var
   generateList: true
+  useGroup: true
   expressionType: resource
   vars:
-    valueIn: ORC.3.1 | OBR.3.1
-    systemCX: ORC.3.2 | OBR.3.2
-    universalId: ORC.3.3 | OBR.3.3
-    universalIdType: ORC.3.4 | OBR.3.4
+    valueIn: ORC.3.1
+    systemCX: ORC.3.2
+    universalId: ORC.3.3
+    universalIdType: ORC.3.4
   constants:
     system: "http://terminology.hl7.org/CodeSystem/v2-0203"
     code: "FILL"
     display: "Filler Identifier"
 
-identifier_3:
+identifier_2b:
+  condition: $valueInORC NULL && $valueIn NOT_NULL
+  valueOf: datatype/Identifier_var
+  generateList: true
+  useGroup: true
+  expressionType: resource
+  vars:
+    valueIn: OBR.3.1
+    systemCX: OBR.3.2
+    universalId: OBR.3.3
+    universalIdType: OBR.3.4
+    valueInORC: ORC.3.1
+  constants:
+    system: "http://terminology.hl7.org/CodeSystem/v2-0203"
+    code: "FILL"
+
+identifier_3a:
   condition: $valueIn NOT_NULL
   valueOf: datatype/Identifier_var
   generateList: true
+  useGroup: true
   expressionType: resource
   vars:
-    valueIn: ORC.2.1 | OBR.2.1
-    systemCX: ORC.2.2 | OBR.2.2
-    universalId: ORC.2.3 | OBR.2.3
-    universalIdType: ORC.2.4 | OBR.2.4
+    valueIn: ORC.2.1
+    systemCX: ORC.2.2
+    universalId: ORC.2.3
+    universalIdType: ORC.2.4
+  constants:
+    system: "http://terminology.hl7.org/CodeSystem/v2-0203"
+    code: "PLAC"
+
+identifier_3b:
+  condition: $valueInORC NULL && $valueIn NOT_NULL
+  valueOf: datatype/Identifier_var
+  generateList: true
+  useGroup: true
+  expressionType: resource
+  vars:
+    valueIn: OBR.2.1
+    systemCX: OBR.2.2
+    universalId: OBR.2.3
+    universalIdType: OBR.2.4
+    valueInORC: ORC.2.1
   constants:
     system: "http://terminology.hl7.org/CodeSystem/v2-0203"
     code: "PLAC"
-    display: "Placer Identifier"
```

Comment:
Why was the display removed?

Reply:
Similarly to [here](https://github.com/CDCgov/prime-reportstream/pull/8017/files/6e7f4a87be69be0f0c71a2cb8d762220b04c1188#r1082811647), I just didn't see any justification for it in the documentation, and it didn't seem to be used in the FHIR->HL7v2 translation.